### ISSUE TITLE
gpth unable to get date for files with parentheses

### ISSUE THREAD

**Issue by casualsailo** [TheLastGimbus/GooglePhotosTakeoutHelper#188]
Hello,

I tried using this tool and found that there was certain files that the program wasn't able to get dates for. Here's a small snippet of the warning messages

> Can't get date on ..\Takeout\Google Photos\Photos from 2019\image(10).jpg
> 
> Can't get date on ..\Takeout\Google Photos\Photos from 2019\image(11).jpg
> 
> Can't get date on ..\Takeout\Google Photos\Photos from 2019\image(12).jpg
> 
> Can't get date on ..\Takeout\Google Photos\Photos from 2019\image(13).jpg
> 
> Can't get date on ..\Takeout\Google Photos\Photos from 2019\image(14).jpg
> 
> Can't get date on ..\Takeout\Google Photos\Photos from 2019\image(15).jpg

When I checked the folder, I noticed the corresponding json file was named differently. The item in parentheses is being placed after the original file extension
> image(10).jpg
> image(11).jpg
> image(12).jpg
> image(13).jpg
> image(14).jpg
> image(15).jpg
> image.jpg(10).json
> image.jpg(11).json
> image.jpg(12).json
> image.jpg(13).json
> image.jpg(14).json
> image.jpg(15).json

If I manually rename the **image.jpg(10).json** file to **image(10).jpg.json**, the program is able to get the date. Would it be possible to update the program to work for file names with parentheses?

**Comment by TheLastGimbus** [comment]
huh... this was carefully observed in #175 

seems like, altough i strip `(n)` in photo names, this does not find the proper json in this case... hmmm

i may add another extractor that will try to link this... i will follow here when i'm done - please be here to test it then ;)

tldr:

> Would it be possible to update the program

yes, i will try :)

**Comment by TheLastGimbus** [comment]
meanwhile, could you help me with testing album support in #187 ? this would speed whole developlent up!

**Comment by casualsailo** [comment]
Thank you for your hard work on this tool. I used your tool once mid 2022 and am glad to see that you are still actively updating it after all this time. I just sent you a small donation!

I personally don't use google photos, but my less tech-savvy family members do. Because google no longer gives free unlimited storage for "high quality", I needed to download the files to free up google storage space

But the downloaded files are a mess! Files don't have the correct date modified, files are scattered in the wrong order in one giant folder (since it's sorted by name instead of date), etc.

Your tool is perfect for the job of organizing the files (using divide-to-dates)! I just wanted to let you know my appreciation for your work

They don't use albums, so I unfortunately can not help you with the testing

**Comment by casualsailo** [comment]
Around 10,000 photos/videos. There are no crashes. The archive/trash folder isn't causing any issues

**Comment by TheLastGimbus** [comment]
> The archive/trash folder isn't causing any issues

and it is there, nicely detected and contains photos that were there?

**Comment by casualsailo** [comment]
I spot checked a few photos, and it does seem like it moved correctly. Files also have their date modified
